README.md

```diff
@@ -123,14 +125,14 @@ bash ./deployment/localup/localup.sh all 1 7
 
 You can use the following approach if you do not know how. 
 
-Run following codes in Greenfield e2e tests to get validator and challenger private keys
-```go
-// please refer to greenfield repo for more information
-// challenger key
-fmt.Println(common.Bytes2Hex(s.Challenger.GetPrivKey().Bytes()))
+```shell
+$ ./build/bin/gnfd keys export validator_bls0 --unsafe --unarmored-hex --home=./deployment/localup/.local/validator0 --keyring-backend=test
```

Comment:
can you change this `/deployment/localup/.local/validator0` to a geeneral one? so is `validator_bls0`

Reply:
Done